swagger.yaml

```diff
@@ -399,6 +485,31 @@ definitions:
         type: string
         description: "Timestamp representation for the publishing process completion, formatted according to RFC3339"
         example: "2020-04-26T10:01:28Z"
+        
+  ImageDownloadUpdate:
+    type: object
+    description: "Subset of ImageDownload variant that can be updated by a caller"
+    properties:
```

Comment:
In the google doc there's mention of a `palette` (eg. "bw") parameter. I intend adding it to my `POST /images/{id}/downloads` task which will follow this so I can add it in or you can add it in here.